### PR TITLE
Fix #9922: Autocomplete ARIA labels

### LINKED ISSUE #9922: AutoComplete: uses hardcoded, english text in widget
https://github.com/primefaces/primefaces/blob/43f22012f9f2bb58ede5bbed83e8652e5b167fbf/primefaces/src/main/resources/META-INF/resources/primefaces/autocomplete/autocomplete.js#L124-L125

### PR DESCRIPTION
**Pull request by melloware**
Fix #9922: Autocomplete ARIA labels

@Rapster need French
@jepsar need Dutch

`primefaces.autocomplete.RESULTS_MESSAGE=results are available, use up and down arrow keys to navigate`

### CODE REVIEW

**Comment by jepsar** [comment]
Ha, I wanted to look into this one this week (back from holidays). One thing: is "results are available ...." prefixed with the number of results? Like "34 results are available ...."? If so, I don't think FR is correct (am I right @Rapster ?), otherwise "results" should start with a capital R.

**Comment by Rapster** [comment]
Exactement! I updated my last coment

File: primefaces/src/main/resources/META-INF/resources/primefaces/autocomplete/autocomplete.js [hunk 1]
```diff
@@ -121,8 +121,7 @@ PrimeFaces.widget.AutoComplete = PrimeFaces.widget.BaseWidget.extend({
         this.cfg.pojo = this.hinput.length == 1;
         this.cfg.minLength = this.cfg.minLength != undefined ? this.cfg.minLength : 1;
         this.cfg.cache = this.cfg.cache || false;
-        this.cfg.resultsMessage = this.cfg.resultsMessage || ' results are available, use up and down arrow keys to navigate';
-        this.cfg.ariaEmptyMessage = this.cfg.emptyMessage || 'No search results are available.';
+        this.cfg.ariaEmptyMessage = this.cfg.emptyMessage;
```
Comment (jepsar) [inline primefaces/src/main/resources/META-INF/resources/primefaces/autocomplete/autocomplete.js]: `this.cfg.resultsMessage` is already used to display the message? (I see we do the fallback in Java now).

**Comment by tandraschko** [comment]
couldnt we use client side localization? this we avoid render text each time for each component

**Comment by melloware** [comment]
> couldnt we use client side localization? this we avoid render text each time for each component

We could but it was already server side localized so I was just updating what was there?

File: primefaces/src/main/resources/META-INF/resources/primefaces/autocomplete/autocomplete.js (hunk 1 above)
Comment (melloware) [reply primefaces/src/main/resources/META-INF/resources/primefaces/autocomplete/autocomplete.js]: yep

**Comment by tandraschko** [comment]
yeah, i know
i was just thinking: f we touch it now, should we optimize it?

just place 3 autoComplete now on the view, we would render 3 times the same msg - instead of using the one from the JS file

**Pull request merged**